.travis.yml

```diff
@@ -13,7 +13,7 @@ before_install:
   - ./run.sh bootstrap
 
 install:
-  - ./run.sh install_aptget  r-cran-rcpp r-cran-runit r-cran-matrix r-cran-pkgkitten r-cran-jsonlite python-scipy
+  - ./run.sh install_aptget  r-cran-rcpp r-cran-runit r-cran-matrix r-cran-pkgkitten r-cran-jsonlite python-scipy r-cran-highlight
```

Comment:
This comingles unrelated issues.

_All_ Suggests are skipped by default.  This worked (for me) for years.

Reply:
Ah, wait, maybe this is different as we didn't use highlight before -- because we also forgot it as vignette engine.